### DISCUSSION TITLE
Normal Tk button color changes when using ttkbootstrap

### DISCUSSION THREAD

**Discussion by Vishwajeet-jr** [israel-dryer/ttkbootstrap#301] [Q&A]
Currently, I am using both the Tk and ttk buttons in my project. The issue is, when using the ttkbootstrap, I am not able to change the basic styling like bg, fg, and active fg color of normal tk buttons. 
as its color gets updated with the default ttkbootstrap styles, is there any possibility to stop this behavior?
The intention to use the normal tk buttons is the UI click effect it provides.
Is there any possibility to get that click effect in our ttkbootstrap? because  when I tried to  pass the extra params, it shows `__init__() takes from 1 to 2 positional arguments but 3 were given` in `ttk.Button`

The extra params are passed as a dict in following format
`BUTTON_BASICS = {
        'relief':"flat",
        'overrelief':"flat",
        'activebackground':"#FFFFFF",
        'cursor':"hand2",
        'foreground':"#000000",
        'background':"#ffffff",
        'font':"-family {Arial Bold} -size 12",
        'borderwidth':"0",
        }`

**Comment by israel-dryer** [comment]
you can remove a tk widget from the ttkbootstrap styling mechanism by passing in the argument `autostyle=False`
https://ttkbootstrap.readthedocs.io/en/latest/styleguide/legacywidgets/

This mean you will be responsible for styling this widget.

As far as ui effects... yes, you can build your own effects by create or subclassing a new style. This is how I created ttkbootstrap. You can see an example of that below. Also check out https://github.com/israel-dryer/ttkbootstrap/discussions/289 to see an example of subclassing a ttkbootstrap style class

https://github.com/israel-dryer/ttkbootstrap/blob/2c6b0bb69820e356deb91d54e408f60e11772a66/src/ttkbootstrap/style.py#L2737-L2756
